prombench/manifests/cluster-infra/2_ingress-nginx-controller.yaml

```diff
@@ -227,7 +263,7 @@ spec:
               add:
                 - NET_BIND_SERVICE
             # www-data -> 33
-            runAsUser: 33
+            runAsUser: 101 # https://github.com/kubernetes/ingress-nginx/issues/4942
```

Comment:
nit: please leave a comment explaining why it's 101

Reply:
Done.